doc/whats-new.rst

```diff
@@ -7270,11 +7270,16 @@ Breaking changes
 - The ``season`` datetime shortcut now returns an array of string labels
   such `'DJF'`:
 
-  .. ipython:: python
-      :okwarning:
+  .. code-block:: ipython
 
-      ds = xray.Dataset({"t": pd.date_range("2000-01-01", periods=12, freq="M")})
-      ds["t.season"]
+      In[92]: ds = xray.Dataset({"t": pd.date_range("2000-01-01", periods=12, freq="M")})
```

Comment:
Should this switch to `ME`? (We'll see in the docs build, so can wait if you're not sure)

Reply:
The "code-block" part and then the ":: ipython" above the code means it is static but with syntax highlighting. [Doc](https://sphinxawesome.xyz/demo/code-blocks/).

I did this as I didn't want to "change history" and my quick fix of :okwarning: was not ok. The code was correct at v0.4 so shouldn't be refactored to run on modern code.